iceoryx2-pal/posix/src/lib.rs

```diff
@@ -77,5 +104,5 @@ pub mod posix {
     #[allow(unused_imports)]
     pub(crate) use common::string_operations::*;
 
-    pub use crate::platform::*;
+    pub use crate::iox2_pal_posix::platform::*;
```

Comment:
Ah maybe the reason is that `pub use crate::platform::platform::*;` reads weirdly?  
Again I vote for the alternative `pub use crate::posix::platform::*;`

Reply:
+1

Reply:
@elfenpiff @orecham How about `crate::os::posix::*` ? 
That could make more sense since we switch between the posix abstractions with `#[cfg(target_os ...`.
A platform is in my understanding more a combination of operating system and hardware.